app/assets/stylesheets/common/base/compose.scss

```diff
@@ -421,10 +421,6 @@ html.composer-open {
     }
   }
 
-  #file-uploader {
-    display: none;
-  }
-
```

Comment:
I'm pretty sure this is just repeating:

https://github.com/discourse/discourse/blob/27b97e4f648da1587e97c5f661da145e12a27bea/app/assets/stylesheets/common/components/pick-files-button.scss

With the downside of hardcoding `#file-uploader`.